src/ShowtimeVerifier.sol

```diff
@@ -23,7 +23,7 @@ contract ShowtimeVerifier is Ownable, EIP712, IShowtimeVerifier {
                             MUTABLE STORAGE
     //////////////////////////////////////////////////////////////*/
 
-    mapping(address => uint256) public signerValidity;
+    mapping(address => uint256) public  ;
```

Comment:
is this ok?

Reply:
i see it was fixed before i submitted hahaha

Reply:
no, t'was a bad commit that I rebased out of existence